src/actions/postEphemeris.ts

```diff
@@ -0,0 +1,35 @@
+'use server';
+
+import { revalidateTag } from 'next/cache';
+import { redirect } from 'next/navigation';
+
+import type { TypeBodyCreateEphemerisV1EphemerisPost, TypeHTTPValidationError, TypeUserIdOut } from '@/__generated__/data-contracts';
+import type { HttpResponse } from '@/__generated__/http-client';
+import Api from '@/libs/Api';
+
+import { REVALIDATION_TAGS } from './tags';
+
+export async function postEphemeris(prevState: any, formData: FormData) {
```

Comment:
`prevState: any`

Do we need this here? It is not used anywhere

Reply:
Oh, I see now. We need this cause formData is passed as a second param.

maybe instead `prevState` use `_prevState`